include.js

```diff
@@ -1,4 +1,11 @@
 let cards = [
+  {
+    artName: 'No Animation',
+    pageLink: './Art/davidr/index.html',
+    imageLink: './Art/vikasverma67/Animation1/lets play ball.gif',
+    author: 'davodr',
+    githubLink: 'https://github.com/0r0ck3t'
+  },
   {
```

Comment:
# Invalid include.js addition
```js
 {
    artName: 'No Animation',
    pageLink: './Art/davidr/index.html',
    imageLink: './Art/vikasverma67/Animation1/lets play ball.gif',
    author: 'davodr',
    githubLink: 'https://github.com/0r0ck3t'
  },
  ```
- folder name invalid - should be named after your github username ( 0r0ck3t )
--> pagelink valid but will need to be reviewed after the folder name change
--> imageLink: you are targetting the link on another contributor and your contribution does not have any pictural representation of your animation : please add your own and refer to the correct path
--> author: looks to have a typo